Misc/NEWS.d/next/Windows/2023-09-29-10-35-29.gh-issue-109991.GmuzGZ.rst

```diff
@@ -0,0 +1,4 @@
+Windows builds now use OpenSSL 1.1.1w. Note that OpenSSL 1.1 has reached its
+end of life and no future fixes will be mode, and this version of Python is
```

Comment:
```suggestion
end of life and no future fixes will be made, and this version of Python is
```

We might also mention that there will be no more installers produced for this version?

Reply:
I thought the wording got a bit awkward when I tried to talk about installers, because anyone can create an installer if they like. Trying to say "the core Python team won't upload new installers to python.org, just like every other time" really didn't work.

Reply:
Fair enough :)